package.json

```diff
@@ -6,32 +6,35 @@
   "author": "The KeystoneJS Development Team",
   "license": "MIT",
   "scripts": {
-    "coverage": "DISABLE_LOGGING=true NODE_ENV=test jest --maxWorkers=1 --coverage",
+    "demo-projects": "yarn --cwd ./demo-projects",
```

Comment:
I think I'd prefer `demo` here, so it's `yarn demo blog start`, which is consistent with the `website`, but wont work for the `test-projects` folder (because there's already a `test` script).

Reply:
Yes I agree. But what to do about `test` I guess. Personally I prefer consistency with `test` over terseness.